css/ie.css

```diff
@@ -1 +1 @@
-.box {behavior: url(js/PIE.htc); position:relative;}
\ No newline at end of file
+.box {behavior: url(js/PIE.htc); position: relative;}
```

Comment:
Add an empty line at the end of the file. (You can set your editor to do this for you)